recipe/emailpassword/emaildelivery/smtpService/serviceImplementation.go

```diff
@@ -0,0 +1,27 @@
+package smtpService
+
+import (
+	"errors"
+
+	"github.com/supertokens/supertokens-golang/ingredients/emaildelivery"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func makeServiceImplementation(config emaildelivery.SMTPServiceConfig) emaildelivery.SMTPServiceInterface {
+	sendRawEmail := func(input emaildelivery.SMTPGetContentResult, userContext supertokens.UserContext) error {
```

Comment:
You need to create an email verification service implementation and use that in the `getContent` function if the input type is email verification email